Misc/NEWS.d/next/Documentation/2024-10-07-00-31-17.gh-issue-125018.yKnymn.rst

```diff
@@ -0,0 +1,4 @@
+The ``importlib.metadata`` documentation now includes semantic
+cross-reference targets for the significant documented APIs. This means
+intersphinx references like :func:`importlib.metadata.version` will
```

Comment:
Did you mean to render the reference or show the syntax? It kinda feels like syntax is that's important:
```suggestion
intersphinx references like

.. code-block:: rst

   :func:`importlib.metadata.version`

will
```

Reply:
This also works:
```suggestion
intersphinx references like ``:func:`importlib.metadata.version``` will
```

Reply:
Really? I remember having problems with trying to do this in RST or Markdown. Nice! TIL.

Reply:
Unfortunately, it doesn't actually work (the sphinx build dies complaining about NEWS containing an inline start literal without an end literal). Even attempting to escape a trailing space to break up the trailing trio of backticks doesn't work (I tried a few different variations on this before giving up).

However, I realised that the NEWS entry containing a working clickable semantics cross-reference to `importlib.metadata.version` illustrated the point just as effectively as quoting the syntax would, so I took the path of least resistance and went with that :)